core/src/main/java/io/seata/core/serializer/SerializerClassRegistry.java

```diff
@@ -0,0 +1,159 @@
+        // register commonly class
+        registerClass(HashMap.class);
+        registerClass(ArrayList.class);
+        registerClass(LinkedList.class);
+        registerClass(HashSet.class);
+        registerClass(TreeSet.class);
+        registerClass(Hashtable.class);
+        registerClass(Date.class);
+        registerClass(Calendar.class);
+        registerClass(ConcurrentHashMap.class);
+        registerClass(SimpleDateFormat.class);
+        registerClass(GregorianCalendar.class);
+        registerClass(Vector.class);
+        registerClass(BitSet.class);
+        registerClass(StringBuffer.class);
+        registerClass(StringBuilder.class);
+        registerClass(Object.class);
+        registerClass(Object[].class);
+        registerClass(String[].class);
+        registerClass(byte[].class);
+        registerClass(char[].class);
+        registerClass(int[].class);
+        registerClass(float[].class);
+        registerClass(double[].class);
+
+        // register seata protocol relation class
+        registerClass(BranchCommitRequest.class);
+        registerClass(BranchCommitResponse.class);
+        registerClass(BranchRegisterRequest.class);
+        registerClass(BranchRegisterResponse.class);
+        registerClass(BranchReportRequest.class);
+        registerClass(BranchReportResponse.class);
+        registerClass(BranchRollbackRequest.class);
+        registerClass(BranchRollbackResponse.class);
+        registerClass(GlobalBeginRequest.class);
+        registerClass(GlobalBeginResponse.class);
+        registerClass(GlobalCommitRequest.class);
+        registerClass(GlobalCommitResponse.class);
+        registerClass(GlobalLockQueryRequest.class);
+        registerClass(GlobalLockQueryResponse.class);
+        registerClass(GlobalRollbackRequest.class);
+        registerClass(GlobalRollbackResponse.class);
+        registerClass(GlobalStatusRequest.class);
+        registerClass(GlobalStatusResponse.class);
+        registerClass(UndoLogDeleteRequest.class);
+        registerClass(GlobalReportRequest.class);
+        registerClass(GlobalReportResponse.class);
+
+        registerClass(MergedWarpMessage.class);
+        registerClass(MergeResultMessage.class);
+        registerClass(RegisterRMRequest.class);
+        registerClass(RegisterRMResponse.class);
+        registerClass(RegisterTMRequest.class);
+        registerClass(RegisterTMResponse.class);
+    }
+    
+    /**
+     * only supposed to be called at startup time
+     *
+     * @param clazz object type
+     */
+    public static void registerClass(Class<?> clazz) {
+        registerClass(clazz, null);
+    }
+
+    /**
+     * only supposed to be called at startup time
+     *
+     * @param clazz object type
+     * @param serializer object serializer
+     */
+    public static void registerClass(Class<?> clazz, Object serializer) {
+        if (clazz == null) {
+            throw new IllegalArgumentException("Class registered to kryo cannot be null!");
```

Comment:
not only for kryo，where  used the  `serializer ` ?

Reply:
received, processed,thx